test/e2e/channle_single_event_test.go

```diff
@@ -1,3 +1,5 @@
+//+build e2e
+
 /*
```

Comment:
filename typo: rename channle in channel

Reply:
Fixed. Thanks.